src/crypto/index.ts

```diff
@@ -231,6 +231,14 @@ export enum CryptoEvent {
     KeyBackupStatus = "crypto.keyBackupStatus",
     KeyBackupFailed = "crypto.keyBackupFailed",
     KeyBackupSessionsRemaining = "crypto.keyBackupSessionsRemaining",
+    /**
+     * Fires when a new valid backup decryption key is in cache.
```

Comment:
It would be nice to write some words about when this happens. As I understand it, it happens when the key is read from secret storage, is received via gossiping, or when a brand new keypair is made (which happens when)?

Reply:
Actually it was only when received via gossip, but it make more sense to make it more global. It's now when the secret is received from another device, from the 4S or when a new one is created (doc udpated)